server/app/rest_api/rest.py

```diff
@@ -49,8 +49,60 @@ def get(self):
         return make_response(jsonify({"schema": current_app.data.schema}), 200)
 
 
+class ConfigAPI(Resource):
+    @swagger.doc({
+        "summary": "Configuration information to assist in front-end adaptation"
+                   " to underlying engine, available functionality, interactive time limits, etc",
+        "tags": ["initialize"],
+        "parameters": [],
+        "responses": {
```

Comment:
this response has a pretty well-defined schema.  is it possible to define it here?

Reply:
Can do a follow up diff if it helps unblock you.

Reply:
Sure, I'm going to postpone this to a new PR though. I'll make a ticket